### PR TITLE
SW Optimizations

### PR DESCRIPTION
**Pull request by jefflarkin**
Optimizations for SW.

### CODE REVIEW

**Comment by atillack** [comment]
@jefflarkin Thank you! I look forward to testing it.

**Review by jeeberhardt — APPROVED** [review]

**Comment by atillack** [comment]
All my tests ran successfully and I do see a speedup of about 10% for SW.

File: cuda/kernel3.cu [hunk 1]
```diff
@@ -28,7 +28,11 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 #define SWAT3 // Third set of Solis-Wets hyperparameters by Andreas Tillack
 
 __global__ void
+#if (__CUDA_ARCH__ == 750)
```
Comment (atillack) [inline cuda/kernel3.cu:31]: Will older compute architectures (like 5.2 or 6.0) also benefit from 1408? Or should this criterion be a bit more inclusive?

**Review by atillack — COMMENTED** [review]
I just tested on a 5.2 GTX Titan X - no speedup but also no slowdown ... so from my end I am happy :-)

**Review by atillack — APPROVED** [review]

**Review by diogomart — APPROVED** [review]
Convergence tests look solid on 36 systems from the set of 42.

**Pull request merged**